cmd/exit_broadcast.go

```diff
@@ -75,66 +108,137 @@ func runBcastFullExit(ctx context.Context, config exitConfig) error {
 		return errors.Wrap(err, "could not load cluster-lock.json")
 	}
 
-	validator := core.PubKey(config.ValidatorPubkey)
-	if _, err := validator.Bytes(); err != nil {
-		return errors.Wrap(err, "cannot convert validator pubkey to bytes")
-	}
-
-	ctx = log.WithCtx(ctx, z.Str("validator", validator.String()))
-
 	eth2Cl, err := eth2Client(ctx, config.BeaconNodeEndpoints, config.BeaconNodeTimeout, [4]byte(cl.GetForkVersion()))
 	if err != nil {
 		return errors.Wrap(err, "cannot create eth2 client for specified beacon node")
 	}
 
-	var fullExit eth2p0.SignedVoluntaryExit
-	maybeExitFilePath := strings.TrimSpace(config.ExitFromFilePath)
-
-	if len(maybeExitFilePath) != 0 {
-		log.Info(ctx, "Retrieving full exit message from path", z.Str("path", maybeExitFilePath))
-		fullExit, err = exitFromPath(maybeExitFilePath)
+	fullExits := make(map[core.PubKey]eth2p0.SignedVoluntaryExit)
+	// multiple
+	if config.All {
+		if config.ExitFromFileDir != "" {
+			entries, err := os.ReadDir(config.ExitFromFileDir)
+			if err != nil {
+				return errors.Wrap(err, "could not read exits directory")
+			}
+			for _, entry := range entries {
+				if !strings.HasPrefix(entry.Name(), "exit-") {
+					continue
+				}
+				exit, err := fetchFullExit(ctx, config.ExitFromFileDir+"/"+entry.Name(), config, cl, identityKey, "")
```

Comment:
`config.ExitFromFileDir+"/"+entry.Name()` is not cross-platform compatible.
Please use `path.Join(config.ExitFromFileDir, entry.Name())` function.